charts/maildev/values.yaml

```diff
@@ -0,0 +1,102 @@
+---
+appName: &name maildev
+
+# Default values for maildev.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: quay.io/jtudelag/maildev
```

Comment:
Is there a reason to not go with [`maildev/maildev`](https://hub.docker.com/r/maildev/maildev) image on Docker Hub? 👀

Reply:
Yes, I could not make it work using env vars....and it is alpine based.

Reply:
hi @jtudelag @ckavili .. just to see whats needed as an alternative - i created a ubi based postfix smtp email relay. correct, it must run under anyuid scc (postfix will only run as root) but maybe a lot simpler to setup ?

https://github.com/eformat/openshift-postfix

thoughts?

Reply:
@eformat yes, we thought about it as it is the default coming in RHEL, but it must run under anyuid as you said. We wanted to avoid that at all cost. 

Also, I would say complexity is more or less the same. For the relay server to be used, you need to configure the server with user & creds, you need a service so the apps can consume it, etc. I don't see any benefits on that side.

Reply:
ubi based is straightforward for sure but I'd rather go with rootless as well 😌 and it'd be great if we can add this chart to maildev repo itself and pull it from there in UJ 🙌🏻

Reply:
Hey @ckavili I will open a PR to maildev repo.

Also, I dont get this: `ubi based is straightforward for sure but I'd rather go with rootless as well`.
What do you mean exactly? I mean, Using ubi and being rootless are two diferent things...

Reply:
Oh I meant the one that Mike has built. It's ubi based and just a `yum install postfix`. Seems simpler but requires privilege :)